### DISCUSSION TITLE
vk::ImageLayout::PRESENT_SRC_KHR missing?

### DISCUSSION THREAD

**Discussion by dawsonc623** [ash-rs/ash#558] [Q&A] [answered]
I am trying to use the constant in the title which is used in the example code. However, I get an error about it not existing, and indeed when I search the repository for this constant the only results are three usages - two in the example code and one in the Debug implementation. Am I dense, or is it missing from the project?

**Comment by Ralith** [comment, marked as answer]
https://github.com/MaikKlein/ash/blob/cebfd544fc609b14646892fc260911d11dac1ea0/ash/src/vk/extensions.rs#L611

**Reply by dawsonc623** [reply]
Thanks!

**Comment by MarijnS95** [comment]
`extensions.rs` is probably "too large" for GitHub and also their new "code search thingy" to index? In any case you can also find out about it in the crate documentation: https://docs.rs/ash/latest/ash/?search=PRESENT_SRC_KHR, that's a bit easier to search than the source.

**Reply by dawsonc623** [reply]
Ah. I see. Searching via docs instead of GitHub is a good call. Thanks!
